tests/common.hpp

```diff
@@ -111,6 +113,15 @@ START_TEST(__testname) {                                                \
 #define BEGIN_CHECK do
 #define END_CHECK while(0)
 
+#define GDR_OPEN_SAFE() ({                                                              \
```

Comment:
can you use an inline function instead of a macro?

Reply:
Fixed.